direct/src/directdevices/DirectJoybox.py

```diff
@@ -3,6 +3,7 @@
 from .DirectDeviceManager import ANALOG_DEADBAND, ANALOG_MAX, ANALOG_MIN, DirectDeviceManager
 from direct.directtools.DirectUtil import CLAMP
 from direct.gui import OnscreenText
+from direct.showbase.ShowBaseGlobal import base
```

Comment:
This will fail, because ShowBaseGlobal doesn't have a `base` member until ShowBase is started.  Better to import ShowBaseGlobal and then in the code use `ShowBaseGlobal.base`.

Reply:
It actually fails already—`DirectJoybox` can't be imported without creating a `ShowBase`. I can still make the changes, though.